File: program 4 sales tax/salesTax.java
// sara kendig 
// this program calculates the sales and local tax in pennsylvania 
// and gives the final sales for the month

import java.util.Scanner;       //import scanner class

public class salesTax {
    public static void main(String[] args){
        double sales;
        double stateTax = .0307;
        double localTax = .06;
        double stateFinal;           // final state total
        double localFinal;           // final local total
        double finalSales;           // finals sales

        // scanner object
        Scanner keyboard = new Scanner(System.in);

        // get sales
        System.out.println("Please enter first value: ");

        sales = keyboard.nextDouble();

        // calculate tax

        stateFinal = stateTax * sales;

        localFinal = localTax * sales;

        finalSales = stateFinal + localFinal + sales;

        // display
        System.out.println("The total sales for the month is $" + sales);
        System.out.println("The state tax for the month is $" + stateFinal);
        System.out.println("The local tax for the month is $" + localFinal);
        System.out.println("The total sales plus tax for the month is $" + finalSales);


    }
}

File: program 4 sales tax/salesTax.cpp
// sara kendig 
// this program calculates the sales and local tax in pennsylvania 
// and gives the final sales for the month

#include<iostream>
#include<iomanip>
using namespace std;

int main() {
	float sales;                 // amount to be taxed
	float state_tax, local_tax;  // tax rates
    double stateFinal;           // final state total
    double localFinal;           // final local total
    double finalSales;           // finals sales
    
	
    state_tax = .0307;
    local_tax = .06;
    
     	// gather our input information
	cout << "Please input the total sales for the month" << endl;
	cin >> sales;
	
	// calculate
    stateFinal = state_tax * sales;

    localFinal = local_tax * sales;

    finalSales = stateFinal + localFinal + sales;

	// output the tax amounts
	cout << fixed << showpoint << setprecision(2)
	     << "The total sales for the month is $" << sales << endl;
	cout << "The state tax for the month is $" << stateFinal << endl;
	cout << "The local tax for the month is $" << localFinal << endl;
    cout << "The total sales plus tax for the month is $" << finalSales << endl;
	
	return 0;
}